src/ModelTraits/SpatieTranslatable/HasTranslations.php

```diff
@@ -65,10 +66,10 @@ public static function create(array $attributes = [])
             if ($model->isTranslatableAttribute($attribute)) { // the attribute is translatable
                 $model->setTranslation($attribute, $locale, $value);
             } else { // the attribute is NOT translatable
-                $model->{$attribute} = $value;
+                $non_translatable[$attribute] = $value;
             }
         }
-        $model->save();
+        $model->fill($non_translatable)->save();
```

Comment:
I don't know if I get you... Do you mean some logic or exception if some non-fillable field is tried to be updated?

The function returns  `true`/`false` to be coherent with the native Laravel update method. So you need to be able to use the same code if you are using or not this trait.